src/main/java/seedu/ui/TextUi.java

```diff
@@ -20,7 +20,13 @@ public TextUi() {
     }
 
     public String getUserInput() {
-        return scanner.nextLine().trim();
+        String userInput = scanner.nextLine().trim();
+        if (userInput.contains(",")) {
+            String newUserInput = userInput.replace(",", "");
```

Comment:
Great call in removing the commas from the user input since they can cause bugs in the Storage File.